src/server/src/routes/facultyMember.ts

```diff
@@ -74,6 +74,30 @@ router.post('/update-profile', async (req: facultyMemberRequest, res: Response)
     }
 });
 
+/******************************************************************************
+ *          GET Request - Read - "GET /api/facultyMember/get-profile"
+ ******************************************************************************/
+
+router.get('/get-profile', async (req: Request, res: Response) => {
+    const { id } = req.body;
```

Comment:
The facultyMemberId can be received from the user object found in the request (as long as the request comes with a cookie, i.e user is authenticated). You can find an example of how to do this in the PR I assigned you, in the file routes/job.ts line 32 - 35.

Reply:
I thought we also want to use this API to let the users look at other people's profiles. (Like when a student wants to look at a faculty member's profile.), so I don't think we always want to get id from the login information. We can create two separate APIs, one that is already implemented and another one that returns the profile of the logged-in user. What do you think?

Reply:
Done.